src/js/hooks/outboundImport/useOutboundImportForm.js

```diff
@@ -166,6 +171,7 @@ const useOutboundImportForm = ({ next }) => {
       // If the save went sucessfully, redirect to the stock movement view page
       window.location = STOCK_MOVEMENT_URL.show(response.data?.data?.id);
     } catch (e) {
+      spinner.hide();
```

Comment:
Is this `hide` only in the catch block? What happens if everything goes well?

Reply:
if everything goes well we are going to get redirected to the show page of the stock movement.

When we call 
```js
window.location = STOCK_MOVEMENT_URL.show(response.data?.data?.id);
```
it still takes some time and if we hide the spinner overlay before redirecting, then user will still be able to click the **finish** button and the issue will not be resolved.
I decided to keep the show overlay until we are redirected, I believe this looks like a natural behavior for the user - it's like after clicking finish we are waiting for the show page to load.

Reply:
I think the `hide` should be called in the finally. Because of redux-persist we might then come back to the React page with the show state of the spinner.

Reply:
`spinner` is blacklisted and is not stored in the redux-presist

Reply:
ok, I was to lazy to check it myself, so thanks for confirming.